Meter.c

```
@@ -505,6 +505,29 @@ int Meter_humanUnit(char* buffer, double value, size_t size) {
    return xSnprintf(buffer, size, "%.*f%c", precision, value, unitPrefixes[i]);
 }
 
+const char* Meter_ioRateUnit(char* buffer, size_t size, double bytesPerSec, bool decimal) {
```

Cppcheck (2.21.0)

[style] The function 'Meter_ioRateUnit' is never used.

(unusedFunction)